app/views/welcome/index.html.erb

```diff
@@ -23,5 +23,19 @@
         <i class="fas fa-user-plus"></i> Local Sign Up
       <% end %>
     </p>
+    <%= form_with url: search_results_path,
+        method: :get do |form| %>
+      <div class="row  mb-6">
+        <div class="col">
+        
+        <%= form.text_field :query, class: "form-control", :placeholder=>"Search...", id: "search"%> 
+        </div>
+        <div class="col-auto">
+        
+        <button type="submit" class = "btn btn-primary"><i class="fa fa-search"></i></button>
```

Comment:
Indent div content again

Reply:
Also the first class should not have spaces around the equals, recommended fix:
`class="btn btn-primary"` instead of `class = "btn btn-primary"`